assets/src/edit-story/output/page.js

```diff
@@ -94,7 +94,7 @@ function OutputPage({ page }) {
           ))}
         </div>
       </amp-story-grid-layer>
-      {ctaElements.length && (
```

Comment:
This can also be `Boolean(ctaElements.length)` (or `!!ctaElements.length`, but I saw that we tend to only use `Boolean`). This way you instantly know that there is only one branch (right now you need to look for the second one just to see that it is null).